readme.md

```diff
@@ -107,6 +120,21 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('songs', function(req, res){
+  res.send('show')
+})
+
+app.post('songs', function(req, res){
+  res.send('create')
+})
+
+app.put('songs', function(req, res){
+  res.send('edit')
+})
+
+app.delete('songs', function(req, res){
+  res.send('destroy')
+})
```

Comment:
We'll want the slash at the beginning of routes unless that's been specified elsewhere